### PR TITLE
fix documentation examples

### LINKED ISSUE #1945: Sorting error using examples from docs
For both `v0.19.3` and `master` on Julia v1.2:

```julia
using DataFrames, CSV
iris = CSV.read(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv"))
sort!(iris)
```
```
ERROR: LoadError: ArgumentError: CSV.Column is read-only; to get a mutable vector, do `copy(col)` or to make all columns mutable do `CSV.read(file; copycols=true)` or `CSV.File(file) |> DataFrame`
Stacktrace:
 [1] setindex!(::CSV.Column{Float64,Float64}, ::Float64, ::Int64) at /home/ian/.julia/packages/CSV/KgT4M/src/tables.jl:26
 [2] permute!!(::CSV.Column{Float64,Float64}, ::Array{Int64,1}) at ./combinatorics.jl:107
```

### PR DESCRIPTION
**Pull request by bkamins**
Fixes #1945

### CODE REVIEW

File: docs/src/man/sorting.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ Sorting is a fundamental component of data analysis. Basic sorting is trivial: j
 ```jldoctest sort
 julia> using DataFrames, CSV
 
-julia> iris = CSV.read(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv"));
+julia> iris = CSV.File(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv")) |>
```
Comment (nalimilan) [inline docs/src/man/sorting.md]: Better use the more common syntax `DataFrame(...)`? That will avoid confusing newcomers.

Comment (bkamins) [reply docs/src/man/sorting.md]: fixed (although in my codes I use the `File` pattern 😄)

Comment (nalimilan) [reply docs/src/man/sorting.md]: No, I meant using `DataFrame(CSV.File(...))` instead of `|>`.

Comment (bkamins) [reply docs/src/man/sorting.md]: OK. Let us stick with `CSV.read` with `copycols=true` (what I pushed now) as it is also a pattern that is good to show I think.

Comment (nalimilan) [reply docs/src/man/sorting.md]: I prefer `DataFrame(CSV.File(...))`, as it makes it less likely that people will call `CSV.read` and forget to pass `copycols=true`. Actually I'd rather get rid of `CSV.read`.

BTW, there are other occurrences in the docs IIRC, better change all of them at the same time.

Comment (bkamins) [reply docs/src/man/sorting.md]: OK - done. At least we agree what should be used 😄.

**Pull request merged**